internal/services/topic.go

```diff
@@ -161,6 +157,11 @@ func (ts *TopicService) reconcileTopic() (TopicReconcileResult, error) {
 				return result, err
 			}
 			glog.Infof("The canary topic %s was created", topicMetadata.Name)
+
+			topicMetadata, err = ts.describeCanaryTopic()
+			if err != nil {
+				return result, err
+			}
```

Comment:
I think this is a side bug fix, right? Before this change, we'll just return empty result even if the topic is created, which is unexpected.

Reply:
No, it is because I want topicMetadata to be populated in the case where the topic has just been created so that I can do `ts.currentLeaders(topicMetadata)` later on in the method.

Reply:
Is that worth a code comment?